Sources/EpoxyCollectionView/CollectionView/Internal/CollectionViewScrollToItemHelper.swift

```diff
@@ -333,21 +322,23 @@ final class CollectionViewScrollToItemHelper {
       forTargetItemIndexPath: targetIndexPath,
       collectionView: collectionView)
 
+    let insetBounds = collectionView.bounds.inset(by: collectionView.adjustedContentInset)
```

Comment:
Before this change this logic would prevent us from scrolling to an item if it was just outside of the visible bounds and obscured by the safe area